runtime-transaction/src/runtime_transaction.rs

```diff
@@ -134,6 +136,74 @@ impl RuntimeTransaction<SanitizedTransaction> {
     }
 }
 
+impl<T: SVMMessage> SVMMessage for RuntimeTransaction<T> {
```

Comment:
🤔 it might actually just be better to impl `AsRef<T>` for `RuntimeTransaction` so we can access the transaction functions directly; regardless of the implementation of this trait.

@tao-stones wdyt?

Reply:
think so, AsRef is lightweight and more flexible while it's being built out. Could impl traits later when things need to be airtight